Sources/Benchmarks/Pose3SLAM.swift

```diff
@@ -49,4 +48,33 @@ let pose3SLAM = BenchmarkSuite(name: "Pose3SLAM") { suite in
       print(val[i, as: Pose3.self].t)
     }
   }
+  
+  var gridDataset =  try! G2OReader.G2ONewFactorGraph(g2oFile3D: try! cachedDataset("pose3example.txt"))
+        // try! G2OReader.G2ONewFactorGraph(g2oFile3D: try! cachedDataset("sphere2500.g2o"))
+  //  check(gridDataset.graph.error(gridDataset.initialGuess), near: 12.99, accuracy: 1e-2)
+  
+  // Uses `NonlinearFactorGraph` on the Intel dataset.
+  // The solvers are configured to run for a constant number of steps.
+  // The nonlinear solver is 5 iterations of Gauss-Newton.
+  // The linear solver is 100 iterations of CGLS.
```

Comment:
This comment talks about a different benchmark.

Reply:
Done